CMakeLists.txt

```diff
@@ -269,7 +269,7 @@ else()
   message(STATUS "resolving Qt version automatically")
 
   while(true)
-    find_package(Qt6 COMPONENTS Core DBus)
+    find_package(Qt6 COMPONENTS Core DBus DBusPrivate)
```

Comment:
Please compare the Qt version first. This private component was split after version 6.10, so it will only be explicitly found if the Qt version is 6.10 or later.

Reply:
Done